apps/blueman-rfcomm-watcher.in

```diff
@@ -18,10 +19,14 @@ def on_file_changed(monitor, file, other_file, event_type):
     if event_type == Gio.FileMonitorEvent.DELETED:
         loop.quit()
 
-mon = Gio.File.new_for_path(sys.argv[1]).monitor_file(Gio.FileMonitorFlags.NONE)
+parser = argparse.ArgumentParser()
+parser.add_argument("path", dest="path", nargs="+", action="store")
```

Comment:
Same here: `nargs="+"` allows multiple values and returns a list, but `os.open` expects a single value.